.github/workflows/ci.yml

```diff
@@ -21,7 +27,6 @@ jobs:
 
       - uses: actions/setup-dotnet@v4
```

Comment:
I think the [duplicate `actions/setup-dotnet@v4` on L26](https://github.com/gsoft-inc/wl-extensions-mongo/blob/63d19a748d72a51fcbfd8b9bfb666059cd77dcd2/.github/workflows/ci.yml#L26C6-L27C1) can be removed?